### ISSUE TITLE
Problem about loading pre-trained weights when fine-tuning.

### ISSUE THREAD

**Issue by CreepyCreeperKree** [jdfxzzy/PEAN#17]
I'm trying to fine-tune on the pre-trained weights 'PEAN_pretrained.pth'. So I followed your instruction:
`python main.py --batch_size="32" --mask --rec="aster" --srb="1" --resume="./ckpt/PEAN_pretrained.pth"`.
But it returned as following:
`Missing key(s) in state_dict: "arm.cnn.conv1.weight", "arm.cnn.conv1.bias", "arm.cnn.conv2.weight", "arm.cnn.conv2.bias", "arm.cnn.batchnorm2.weight", "arm.cnn.batchnorm2.bias", "arm.cnn.batchnorm2.running_mean", "arm.cnn.batchnorm2.running_var", "arm.cnn.conv3.weight", "arm.cnn.conv3.bias", "arm.cnn.conv4.weight", "arm.cnn.conv4.bias", "arm.cnn.batchnorm4.weight", "arm.cnn.batchnorm4.bias", "arm.cnn.batchnorm4.running_mean", "arm.cnn.batchnorm4.running_var", "arm.cnn.conv5.weight", "arm.cnn.conv5.bias", "arm.cnn.conv6.weight", "arm.cnn.conv6.bias", "arm.cnn.batchnorm6.weight", "arm.cnn.batchnorm6.bias", "arm.cnn.batchnorm6.running_mean", "arm.cnn.batchnorm6.running_var", "arm.rnn.0.rnn.weight_ih_l0", "arm.rnn.0.rnn.weight_hh_l0", "arm.rnn.0.rnn.bias_ih_l0", "arm.rnn.0.rnn.bias_hh_l0", "arm.rnn.0.rnn.weight_ih_l0_reverse", "arm.rnn.0.rnn.weight_hh_l0_reverse", "arm.rnn.0.rnn.bias_ih_l0_reverse", "arm.rnn.0.rnn.bias_hh_l0_reverse", "arm.rnn.0.embedding.weight", "arm.rnn.0.embedding.bias", "arm.rnn.1.rnn.weight_ih_l0", "arm.rnn.1.rnn.weight_hh_l0", "arm.rnn.1.rnn.bias_ih_l0", "arm.rnn.1.rnn.bias_hh_l0", "arm.rnn.1.rnn.weight_ih_l0_reverse", "arm.rnn.1.rnn.weight_hh_l0_reverse", "arm.rnn.1.rnn.bias_ih_l0_reverse", "arm.rnn.1.rnn.bias_hh_l0_reverse", "arm.rnn.1.embedding.weight", "arm.rnn.1.embedding.bias".`.
It seems that the pre-trained weights lacks the weights of ARM. So should I turn `strict=False` when loading? Or I missed anything?

**Comment by YongshunWang** [comment]
请问解决了吗
